transformer_engine/pytorch/module/layernorm_linear.py

```diff
@@ -696,13 +696,10 @@ def __init__(
         self.reset_layer_norm_parameters()
 
         if not skip_weight_param_allocation:
-            self.register_buffer("weight_tensor",
-                                 torch.empty(
-                                    self.out_features,
-                                    self.in_features,
-                                    device=torch.cuda.current_device(),
-                                    dtype=params_dtype),
-                                 persistent=False)
+            self.weight_tensor = torch.empty(
```

Comment:
This still creates a tensor that won't be sharded by FSDP because it doesn't recognize it as a parameter. How do you address that?